src/usdb_syncer/gui/mw.py

```diff
@@ -269,6 +272,83 @@ def _generate_song_pdf(self) -> None:
         if path:
             generate_song_pdf(self.table.all_local_songs(), path)
 
+    def _import_usdb_ids_from_files(self) -> None:
+        logger = logging.getLogger()
```

Comment:
oh oops, that was new for me, I will change that

Reply:
I am so sorry. 3 weeks ago I was not able to find any use of a `get_logger`-method in the project so I thought you mean the `__file__` part. I don't know why I could not find it, because on the 2nd glance yesterday I easily found it. But then I was not sure if you really mean `global` like outside the class, because I saw we used `self.logger = get_logger(__file__)` as instance attribute somewhere else. Hence I decided to do so too 🙈 

My fault, I will change it to what you actually already suggested in the very first comment.